stdlib/EvaluatorSettings.pkl

```diff
@@ -115,6 +115,10 @@ class Http {
   proxy: Proxy?
 }
 
+/// Whether to pretty print calls to trace(), producing indented output across multiple lines.
+@Since { version = "0.28.3" }
```

Comment:
A change like this will land in a minor release, not a patch.
```suggestion
@Since { version = "0.29.0" }
```

Reply:
Updated to `0.29.0`.